### PR TITLE
fix: broken conversion to LdesFragmentIdentifier

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldes-server-application/pom.xml [hunk 1]
```diff
@@ -130,13 +130,23 @@
             </dependencies>
         </profile>
         <profile>
-            <id>fragmentation-pagination</id>
```
Comment (Tomvbe) [inline ldes-server-application/pom.xml:133]: why are you removing pagination?

File: ldes-server-application/pom.xml [hunk 2]
```diff
@@ -130,13 +130,23 @@
...
+            <id>fragmentation</id>
             <dependencies>
+                <dependency>
+                    <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+                    <artifactId>ldes-fragmentisers-common</artifactId>
```
Comment (Tomvbe) [inline ldes-server-application/pom.xml:137]: shouldn't this be added to all fragmentation profiles?

File: ldes-server-application/pom.xml [hunk 3]
```diff
@@ -130,13 +130,23 @@
...
+                    <version>${project.version}</version>
+                </dependency>
                 <dependency>
                     <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
                     <artifactId>ldes-fragmentisers-pagination</artifactId>
                     <version>${project.version}</version>
                 </dependency>
+                <dependency>
```
Comment (Tomvbe) [inline ldes-server-application/pom.xml:145]: Shouldn't this be added to al fragmentation profiles?

**Comment by Tomvbe** [comment]
Can we add a test that fails without this fix and succeeds with this fix? Somewhere in the integration test layer of the mongo repository?

File: ldes-server-application/pom.xml (hunk 2 above)
Comment (Yalz) [reply ldes-server-application/pom.xml:137]: this actually isn't needed. since there already is a dependency in each module to common

**Review by Tomvbe — APPROVED** [review]

**Pull request merged**